package.xml

```diff
@@ -1,36 +1,28 @@
 <?xml version="1.0"?>
-<package>
+<package format="3">
   <name>steering_functions</name>
-  <version>0.2.0</version>
+  <version>0.3.0</version>
   <description>The steering_functions package</description>
 
   <maintainer email="holger.banzhaf@de.bosch.com">Holger Banzhaf</maintainer>
   <author email="holger.banzhaf@de.bosch.com">Holger Banzhaf</author>
 
   <license>Apache-2.0</license>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend condition="$ROS_VERSION == 1">catkin</buildtool_depend>
+  <buildtool_depend condition="$ROS_VERSION == 2">cmake</buildtool_depend>
 
-  <build_depend>roscpp</build_depend>
-  <build_depend>geometry_msgs</build_depend>
-  <build_depend>nav_msgs</build_depend>
-  <build_depend>visualization_msgs</build_depend>
-  <build_depend>roslib</build_depend>
-  <build_depend>costmap_2d</build_depend>
-  <build_depend>tf</build_depend>
-  <build_depend>cmake_modules</build_depend>
-  <build_depend>eigen</build_depend>
+  <build_depend>ros_environment</build_depend>
 
-  <run_depend>roscpp</run_depend>
-  <run_depend>geometry_msgs</run_depend>
-  <run_depend>nav_msgs</run_depend>
-  <run_depend>visualization_msgs</run_depend>
-  <run_depend>roslib</run_depend>
-  <run_depend>costmap_2d</run_depend>
-  <run_depend>tf</run_depend>
-  <run_depend>eigen</run_depend>
+  <depend condition="$ROS_VERSION == 1">costmap_2d</depend>
+  <depend>eigen</depend>
```

Comment:
Can we move this one line up?

Reply:
Sure thing, the idea was to be alphabetical